app/pathogen/arbovirus/dashboard/(map)/MapArbovirusFilter.tsx

```diff
@@ -1,12 +1,13 @@
-import { ArboActionType, ArboContextType } from "@/contexts/arbo-context/arbo-context";
 import { Checkbox } from "@/components/ui/checkbox";
 import { pathogenColorsTailwind } from "./ArbovirusMap";
 import { Button } from "@/components/ui/button";
 import SectionHeader from "@/components/customs/SectionHeader";
+import { PathogenContextType, PathogenContextActionType } from "@/contexts/pathogen-context/pathogen-context";
+import { ArbovirusEstimate } from "@/contexts/pathogen-context/pathogen-contexts/arbo-context";
 
 interface MapArbovirusFilterProps {
   className?: string;
-  state: ArboContextType;
+  state: PathogenContextType<ArbovirusEstimate>;
```

Comment:
Remember `ArbovirusEstimate` is an `any` under the hood. This is just so that if we eventually fix the types we'll get type errors in the right places and know which `any`s to change.